### PR TITLE
chore: update FluentUI to v8

### PR DESCRIPTION
**Pull request by OEvgeny**
## Description

Update Fluent UI to the latest version:
- Updated all projects to use `@fluentui/react@^8.57.1` and dependent packages
- Changed all imports to `@fluentui/react`
- Updated eslint rule to use `@fluentui/react` package name for deep imports enforcement
- Updated jest configurations and fixed failing tests
- Fixed lint issues
- Updated `yarn.lock` and `yarn-berry.lock` for all projects

## Task Item

To be added

## Screenshots

image
image

#minor

### CODE REVIEW

File: Composer/packages/lib/code-editor/src/lu/__tests__/DefineEntityButton.test.tsx [hunk 1]
```diff
@@ -67,11 +64,6 @@ describe('<DefineEntityButton />', () => {
       jest.runAllTimers();
     });
 
-    await act(async () => {
-      fireEvent.click(screen.getByText('this page'));
-    });
-
-    expect(window.open).toBeCalled();
-    window.open = origOpen;
+    expect(screen.getByText('this page')).toBeInTheDocument();
```
Comment (OEvgeny) [inline Composer/packages/lib/code-editor/src/lu/__tests__/DefineEntityButton.test.tsx:67]: Fluent doesn't call `window.open` after it got updated. Manually verified the link does work:
image

File: Composer/package.json [hunk 2]
```diff
@@ -135,7 +135,6 @@
     "eslint": "7.0.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-formatter-github-actions": "^1.0.0",
-    "eslint-plugin-security": "^1.4.0",
```
[the remark this answers is not in the document]
Comment (OEvgeny) [reply Composer/package.json:138]: image

File: Composer/packages/client/src/components/CreationFlow/AzureBotDialog.tsx [hunk 3]
```diff
@@ -90,7 +90,7 @@ export const AzureBotDialog = (props: Props) => {
         defaultSelectedKey="Create"
         options={options}
         styles={{
-          applicationRole: {
+          root: {
```
[the remark this answers is not in the document]
Comment (OEvgeny) [reply Composer/packages/client/src/components/CreationFlow/AzureBotDialog.tsx:93]: image

**Review by cwhitten — APPROVED** [review]

**Review by tonyanziano — APPROVED** [review]
Awesome, thanks!

**Pull request merged**